bundles/AdminBundle/src/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -1420,7 +1420,10 @@ public function doExportAction(Request $request, LocaleServiceInterface $localeS
         return $this->adminJson(['success' => true]);
     }
 
-    public function encodeFunc($value): string
+    /**
+     * @param array{fieldName: string, data: string} $value
+     */
+    public function encodeFunc(array $value): string
```

Comment:
@blankse shouldn´t it be `string $value` ? See phpstan error here https://github.com/pimcore/pimcore/actions/runs/3890921482/jobs/6640541602 , what do you think?

Reply:
@mcop1 It was change here to an array: https://github.com/pimcore/pimcore/pull/13889

Reply:
@mcop1 When you reverted this, than you need to change it to a string, right.